### ISSUE TITLE
Feature request: Support downloading Scenarist and TTML captions

### ISSUE THREAD

**Issue by Nii-90** [rg3/youtube-dl#3432]
I don't know if this is a globally-set option or needs to be added per-site, but I've currently seen it on Comedy Central.

Even though the 400 Request error on issue #3364 is still a problem for atmidnight and such (if the RTMP workaround isn't applicable), I noticed that subtitles were included in the segment pages output by the --write-pages option, but that these are not saved by youtube-dl.

The two subtitle formats it carries are Scenarist's Closed Caption format (.scc) and the XML-based TTML format (.xml).

There doesn't seem to be any obfuscating of the URLs, as I was able to grab both of the files with a plain invocation of wget.  It'd just be kind of a burden to have to parse the sub locations out of the ```*.dump``` files and download them separately.

While I'm not currently aware of any support for these formats in VSFilter, et al., there are utilities that can convert them to SRT or ASS, which would make them useful to save if the user requests subtitles be saved.

For instance, this is how they show up in the segment pages:
```
<?xml version="1.0" encoding="UTF-8" ?> 
<package version="1.7.1">
...snip...
      <transcript kind="captions" srclang="en" label="English">
        <typographic format="cea-608" src="http://a6.akadl.mtvnservices.com/9950/mtvnorigin/gsp.comedystor/com/at_midnight/Season_1/episode_01092/am_01092_ex_act1.scc"/>
        <typographic format="ttml" src="http://a6.akadl.mtvnservices.com/9950/mtvnorigin/gsp.comedystor/com/at_midnight/Season_1/episode_01092/am_01092_ex_act1.dfxp.xml"/>
      </transcript>
...snip...
  </video>
</package>
<!-- feeds/sync/mediagen -->
```

**Comment by yan12125** [comment]
I've added an experimental implementation in bf6427d2fbcbd95cd1cb640e8b894c18782a2a12. Now TTML/dfxp subtitles are supported.

**Comment by dstftw** [comment]
This has been implemented in youtube-dl 2015.04.26. See our [FAQ](https://yt-dl.org/update) if you have problems updating.
